app/components/Footer.tsx

```diff
@@ -1,34 +1,13 @@
-import { Link, Form } from "@remix-run/react";
-import PrimaryButton from "./buttons/PrimaryButton";
-
 interface FooterProps {
   isAuthenticated: boolean;
 }
 
 const Footer = ({ isAuthenticated }: FooterProps) => {
   return (
-    <footer className="footer flex justify-center">
+    <footer className="footer mt-8 flex justify-center px-[1rem] sm:mx-[1rem]">
```

Comment:
Her bør vi fjerne `sm:mx-[1rem]` fordi det gjør at man får en uønsket kant på bakgrunnsbildet til footeren: 

<img width="1681" alt="Skjermbilde 2023-05-11 kl  09 58 06" src="https://github.com/varianter/joy/assets/69193134/d4389c7b-0850-44ae-8e04-9972cc55411b">

Reply:
hvis en bruker padding isteden så vil ikke bildet croppes

Reply:
Jeg tror også vi kan øke `px-[1rem]` for å få litt mer padding på sidene på mobilvisning